model/game.js

```diff
@@ -1,5 +1,6 @@
 const { Model, DataTypes } = require('sequelize');
 const { getDbLazy } = require('../repository/database');
+const { GameMode } = require('./gameMode');
```

Comment:
Er du sikker på at denne importen eigentleg fungerar (same gjelder import av Game i gameMode.js)?
For når du brukar `module.exports = Game` så skal ein eigentleg bruke `const Game = require('./game');`.

Reply:
den klager ikke? men nei, vet ikke